tests/scripts/getdefines.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+"""Extract defines create by header
```

Comment:
I'm not convinced by this approach. It seems a little roundabout and fragile. If `config_psa.h` is missing a `#define` that it should have, the tests won't catch it.

`scripts/config.py` already knows which symbols in `config.h` are crypto. How about adding a command that turns off all crypto symbols?